src/common/libjob/Makefile.am

```diff
@@ -54,7 +54,6 @@ test_ldadd = \
         $(top_builddir)/src/common/libtap/libtap.la \
```

Comment:
Commit message:   "macro was more than necessary'.  Missing "used"?

That was commit 6234eb20fae7ced9ada4c22ddf76a7d6230c7afa